### PR TITLE
Storybook: Fix checkbox spacing

### LINKED ISSUE #27065: Storybook: Fix checkbox spacing
Compared to the other form elements the label and description text have a different line-height so it’s spaced differently. They inherit body 1.5 line-height, the other components use a 1.25 line-height. 

And we have a `<br>` in between, which is not required.

### PR DESCRIPTION
**Pull request by Lundbyit**
**What this PR does / why we need it**:
* Remove unecessary <br>
* add line-height to description to align it with other form component styles

**Which issue(s) this PR fixes**:
Suggestion on how to fix #27065

Fixes #27065 

**Special notes for your reviewer**:
Not sure if the line-height should be in the checkbox description styles or in label description styles.

### CODE REVIEW

File: packages/grafana-ui/src/components/Forms/Checkbox.tsx [hunk 1]
```diff
@@ -25,6 +25,7 @@ export const getCheckboxStyles = stylesFactory((theme: GrafanaTheme) => {
     description: cx(
       labelStyles.description,
       css`
+        line-height: 1.25px;
```
Comment (Clarity-89) [inline packages/grafana-ui/src/components/Forms/Checkbox.tsx:28]:
```suggestion
        line-height: 1.25;
```

**Review by Clarity-89 — CHANGES_REQUESTED** [review]
Thank you for your contribution! 🎉   It looks good, just one minor thing to fix: we use unitless values for line height instead of pixels.

File: packages/grafana-ui/src/components/Forms/Checkbox.tsx (hunk 1 above)
Comment (peterholmberg) [reply packages/grafana-ui/src/components/Forms/Checkbox.tsx:28]: was thinking if we should use the line heights in the `theme.typography.lineHeight` ?

Comment (Clarity-89) [reply packages/grafana-ui/src/components/Forms/Checkbox.tsx:28]: We don't have `line-height` 1.25 in the themes, the closest is 1.1, which also works I think.

Comment (peterholmberg) [reply packages/grafana-ui/src/components/Forms/Checkbox.tsx:28]: I'd suggest using that so we don't introduce custom line-heights

Comment (Lundbyit) [reply packages/grafana-ui/src/components/Forms/Checkbox.tsx:28]: Thanks for the pointer. I'll fix that

**Review by Clarity-89 — APPROVED** [review]
Great job!

**Comment by Lundbyit** [comment]
Thanks! :) And thanks for the help with this PR

**Pull request merged**